htdocs/router.php

```diff
@@ -27,7 +27,8 @@
 if (preg_match(
     '#^([a-zA-Z_-]+)/ajax/([a-zA-Z0-9_.-/]+)$#',
     $url
-)) {
+)
```

Comment:
This new coding change is ugly.

Reply:
I think it's a bit messy either way... the regex could be in it's own variable.

Reply:
I'm happy to be in javascript land and not requiring this change because it just seems like waste of lines.

Reply:
Likewise -- when working with it I feel like Javascript code is a waste of lines :D

Reply:
For real though I think this change indicates bad code smells/style. It does look ugly but multi-line if statements are generally pretty ugly anyway.